content/open-source/accoutrement.rst

```diff
@@ -0,0 +1,96 @@
+public: yes
```

Comment:
`/accoutrement/` is legacy. The new pattern means we want `/open-source/accoutrement/` to be the canon. I'm just worried about breaking old links. Maybe there's a better way?